package.json

```diff
@@ -40,15 +40,17 @@
   "scripts": {
     "build:site": "cd site && npm ci && npm run build",
     "start": "node ./bin/run",
-    "test": "run-s test:*",
-    "ava": "ava --verbose",
-    "format": "prettier --write \"{src,scripts,site,site/src,tests,.circleci,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\"",
-    "test:lint": "eslint --fix \"{.github,src,scripts,site,tests}/**/*.{js,md,html}\"",
-    "test:format": "prettier --check \"{src,scripts,site,site/src,tests,.circleci,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\"",
```

Comment:
`prettier --write` won't fail the script if the formatting is not valid (as opposed to `--check`). That can happen if someone edits the code directly on GitHub or disables commits hooks.

Maybe we can use package json variables to share the configuration?

This is also relevant for `eslint --fix` (that was already an issue with the previous scripts).

Reply:
I though `prettier --write` exit code was `1` if a file was modified, but you're right, this is not the case.
`eslint --fix` does give an exit code that is `1` on linting errors, but not if the error was autofixed, so you're right on this one too.

I will fix this.

Could you please describe what you mean by `package.json` variables to share the configuration?

Reply:
Can we do something like this:
```json
{
    "name"   : "myapp",
    "prettier" : { "files" : "...." },
    "scripts": {
        "format:write" : "prettier --write $npm_package_prettier_files",
        "format:check" : "prettier --check $npm_package_prettier_files"
    },
    ...
}
```

Haven't tested that one, but it should work

Reply:
Great solution! Yes, this should work, let's try it.

Reply:
Fixed.